app/controllers/ping_controller.rb

```diff
@@ -1,5 +1,9 @@
 class PingController < ApplicationController
   def index
-    render :text => 'pong', :status => 200
+    if ActiveRecord::Base.connection.active?
+      render :text => 'pong', :status => 200
+    else
+      raise PG::Error
```

Comment:
Is this needed?  `.connection` or `active?` should raise if PG goes down or is otherwise unavailable.

Reply:
Sorry, active? won't raise but connection might if the connection isn't already established.

Reply:
Nope, `.connection` doesn't raise anything after a disconnect happens.

Reply:
👍